### ISSUE TITLE
How do you even use this?

### ISSUE THREAD

**Issue by TobiTenno** [s-yadav/FlakeId#2]
I'm trying to include it in-browser.
I downloaded the minified file, included it in my file header, and tried to call 
`new FlakeId().gen()`

but I get

```
Uncaught TypeError: FlakeId is not a constructor
```

How am I supposed to use it?

**Comment by s-yadav** [comment]
Hummm, I guess I have added wrong bundling config. I will fix this. For now you can use 
`new FlakeId.default()`

**Comment by TobiTenno** [comment]
That still doesn't work

```bash
ReferenceError: c is not defined
    at FlakeId.gen (...\node_modules\flakeid\lib\flakeid.js:68:12)
```

**Comment by TobiTenno** [comment]
@s-yadav i added the error output

**Comment by s-yadav** [comment]
I will fix this

**Comment by s-yadav** [comment]
Fixed it on v0.3.0. Let me know if you still face any issue

**Comment by TobiTenno** [comment]
I switched to a different flake Id generator this morning, I'll still give it a shot later, however.

**Comment by TobiTenno** [comment]
worked for me. thanks.

**Comment by pavanSaberjack** [comment]
@s-yadav  Still facing same error in 0.4.0 , Do we need use Flake.default in new versions ? If yes please update the documentation.

**Comment by s-yadav** [comment]
Fixed this on 1.0.0. Now you don't have to use default property. 
```js
const FlakeId = require('flakeid'); //this will work.
```
